lib/vsc/utils/mail.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, mail_host=None, mail_to=None, mail_from=None, mail_subject=No
         """Initialisation.
 
         @type mail_host: string
-        @type mail_to: string
+        @type mail_to: string or list of strings
```

Comment:
is this a breaking change or can it never have worked with a list?

Reply:
rephrase as `list of strings (or string, but deprecated)` or something